app/invocation/invocation_targets.tsx

```diff
@@ -93,6 +93,19 @@ export default class TargetsComponent extends React.Component {
             pageSize={this.props.pageSize}
           />
         )}
+
+        {!!this.props.model.skipped.length && (
+          <TargetsCardComponent
+            buildEvents={this.props.model.skipped}
+            className="card-skipped"
+            iconPath="/image/info.svg"
```

Comment:
Should we use this `skip-forward` icon? https://feathericons.com/?query=skip-forward